docs/formats/open_images_user_manual.md

```diff
@@ -15,7 +15,25 @@ on [its website](https://storage.googleapis.com/openimages/web/download.html).
 Datumaro supports versions 4, 5 and 6.
 
 Datumaro currently supports only the human-verified image-level label
-annotations from this dataset.
+annotations and bounding box annotations from this dataset.
+
+One attribute is supported on the labels:
+
+- `score`: the confidence level from 0 to 1. A score of 0 indicates that
```

Comment:
```suggestion
- `score` (float): the confidence level from 0 to 1. A score of 0 indicates that
```

Reply:
OK, updated all attribute descriptions to be consistent with that format.